lib/xliffer/xliff/string.rb

```diff
@@ -41,6 +49,10 @@ def trans_unit?(xml)
         xml.name.downcase == 'trans-unit'
       end
 
+      def text_node_to_components(node)
+        node.children.map { |cell| [cell.name, cell.text] }
```

Comment:
It would be nicer to create our own type with explicit properties.

Reply:
I tried this in 491d16b  What do you think?

Reply:
Looks good, thanks.